.github/workflows/ci.yml

```diff
@@ -94,7 +94,7 @@ jobs:
           run: |
             cd bor
             git submodule update --init --recursive --force
-            git fetch --no-tags --prune --depth=1 origin +refs/heads/master:refs/remotes/origin/master
+            git fetch --no-tags --prune --depth=1 origin +refs/heads/develop:refs/remotes/origin/develop
```

Comment:
can we give it a try with `master` too?
I'd prefer having the default branch here, thanks